demos/super_resolution_demo/main.cpp

```diff
@@ -136,6 +136,10 @@ int main(int argc, char *argv[]) {
             int h = static_cast<int>(lrInputInfoItem->getTensorDesc().getDims()[2]);
             int c = static_cast<int>(lrInputInfoItem->getTensorDesc().getDims()[1]);
 
+            /** Convert to 3-channels image for case when image was saved with alpha channel **/
+            if (img.channels() == 4)
+                cv::cvtColor(img, img, cv::COLOR_BGRA2BGR);
```

Comment:
Wouldn't it be better to change `IMREAD_UNCHANGED` to `IMREAD_COLOR` above? It's not only simpler, but would also work for grayscale images.

Reply:
No. 'IMREAD_COLOR' always convert to RGB Image, but text-image-super-resolution model expect grayscale Image as input.

Reply:
I see. In this case, maybe you could use `IMREAD_COLOR`/`IMREAD_GRAYSCALE`, depending on `c`?

Reply:
Maybe

Reply:
@IRDonch  is it possible add testcase for grayscale input image?

Reply:
Not at the moment, I'm afraid. I don't think we have any in our test data.